src/request.ts

```diff
@@ -73,6 +76,11 @@ const attachErrorDetails = (err: unknown) => {
   throw err
 }
 
+export const addVersionToUrl = (url: string, version: Version = "v3"): string => {
```

Comment:
Are we already accepting v3 as a version for api gateway?

Reply:
Aye i think this commit: https://bitbucket.org/yourwealth/api-gateway/commits/89b5ea4334686d8cc237f8bc3a304688c9274e89